spectrum/src/main/res/values/attrs.xml

```diff
@@ -1,14 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <attr name="spectrum_colors" format="reference" />
+    <attr name="spectrum_strokeWidth" format="dimension" />
```

Comment:
I'd consider renaming this `spectrum_borderWidth` or `spectrum_borderStrokeWidth`. I think it's important that it be self-evident that this applies to the border around the color.